libs/blocks/tabs/tabs.css

```diff
@@ -358,7 +381,11 @@
   border: none;
 }
 
-.tabs.radio .tab-paddles {
+.tabs.radio .paddle-left {
```

Comment:
can't we do `.tabs.radio .paddle { display: none }` here? It would target both `paddle-left` and `paddle-right` elements